File: task2.js
function Count_Digit(Num) {
    let digit_len = Math.abs(Num).toString().length;
    if (digit_len % 2 !== 0) {
        return 1;
    } else {
        return 0;
    }
}

let entry_num = prompt("Enter the numbers with a space: ");
let Nums = entry_num.split(" ").map(num => parseInt(num));
let count_odd_digits = Nums.reduce((acc, num) => acc + Count_Digit(num), 0);
console.log(count_odd_digits);

File: task2.swift
import Foundation

func countDigit(_ num: Int) -> Int {
    let digitLen = String(abs(num)).count
    if digitLen % 2 != 0 {
        return 1
    } else {
        return 0
    }
}

print("Enter the numbers with a space:")
let entryNum = readLine()
if let entryNum = entryNum {
    let nums = entryNum.split(separator: " ").compactMap { Int($0) }
    let countOddDigits = nums.reduce(0) { $0 + countDigit($1) }
    print(countOddDigits)
}
